_episodes/07-find.md

```diff
@@ -382,6 +382,65 @@ Miscellaneous:
 > {: .solution}
 {: .challenge}
 
+> ## Using grep in a loop
+> Navigate to Nelle's `data-shell/data/pdb/` directory, which contains Protein Data Bank files.
+> Write a `for` loop which uses `echo` and `grep` to print each file name followed by the number
+> of atoms in the compound.
+>
+> Here's the beginning and end of the output your loop should return:
+> ```
+> aldrin.pdb 26
+> ammonia.pdb 4
+> ascorbic-acid.pdb 20
+> ...
+> vanillin.pdb 19
+> vinyl-chloride.pdb 6
+> vitamin-a.pdb 51
+> ```
+> {: .output}
+>
+> Hint: Read `man echo` for an option to suppress newlines
+> and `man grep` for an option to produce counts only.
+>
+> > ## Solution
+> > ```
+> > for file in *.pdb
+> >	do
+> >		echo -n $file " "
+> >		grep -c ATOM $file
+> >	done
```

Comment:
It's possible to get functionally the same output with just `grep -c 'ATOM' *.pdb` (the only difference is that the name and counts are separated by colons rather than spaces). That makes this example a little less compelling to me.